src/cloudai/workloads/ai_dynamo/slurm_command_gen_strategy.py

```diff
@@ -27,10 +27,33 @@ class AIDynamoSlurmCommandGenStrategy(SlurmCommandGenStrategy):
 
     def _container_mounts(self) -> list[str]:
         td = cast(AIDynamoTestDefinition, self.test_run.test.test_definition)
+
+        dynamo_repo_path = td.dynamo_repo.installed_path
```

Comment:
Instead of having an always-mounted repo, perhaps we should make it optional. Mount it if provided in toml.

Reply:
And while we are at it, would this mechanism allow to provide a locally cached git repo (with local changes perhaps)? This would be helpful during dev/debug cycle.